server/asset/btc/btc.go

```diff
@@ -420,8 +414,8 @@ func (btc *Backend) transaction(txHash *chainhash.Hash) (*Tx, error) {
 		})
 	}
 	var feeRate uint64
-	if verboseTx.Size > 0 {
-		feeRate = (sumIn - sumOut) / uint64(verboseTx.Size)
+	if verboseTx.Vsize > 0 {
+		feeRate = (sumIn - sumOut) / uint64(verboseTx.Vsize)
```

Comment:
`Vsize` is foreign to me, and it's not clear to me that we want to use it.

```go
// GetTxVirtualSize computes the virtual size of a given transaction. A
// transaction's virtual size is based off its weight, creating a discount for
// any witness data it contains, proportional to the current
// blockchain.WitnessScaleFactor value.
func GetTxVirtualSize(tx *btcutil.Tx) int64 {
	// vSize := (weight(tx) + 3) / 4
	//       := (((baseSize * 3) + totalSize) + 3) / 4
	// We add 3 here as a way to compute the ceiling of the prior arithmetic
	// to 4. The division by 4 creates a discount for wit witness data.
	return (blockchain.GetTransactionWeight(tx) + (blockchain.WitnessScaleFactor - 1)) /
		blockchain.WitnessScaleFactor
}
```

What's the reasoning for "virtual size" here?

Reply:
Since segwit, the maximum block size is measured in weight units, or alternatively vsize = weight/4. This means that the miners don't look at the byte size when considering whether to include a transaction, they look at the weight.